crates/node/p2p/src/lib.rs

```diff
@@ -13,8 +13,8 @@ extern crate tracing;
 mod metrics;
 pub use metrics::Metrics;
 
-mod net;
-pub use net::{Broadcast, Config, Network, NetworkBuilder, NetworkBuilderError};
+mod config;
+pub use config::Config;
```

Comment:
Does it make sense to have the config live anymore in `kona_p2p` if it's only used by the `NetworkDriver` which is now moved into the `kona-node-service` actors module?